Vagrantfile

```diff
@@ -53,6 +53,7 @@ Vagrant.configure(2) do |config|
       ansible-playbook -u vagrant -i 'localhost,' --extra-vars "aws_profile=raster-foundry" \
           raster-foundry.yml
       cd /opt/raster-foundry
+      export AWS_PROFILE=raster-foundry
```

Comment:
This is so that the `aws s3 cp` has the profile before we get a chance to set it globally within the VM using Ansible?